### PR TITLE
requestTemperaturesByAddress fixed to work in parasite power mode

### PR DESCRIPTION
**Pull request by pezinek**
Apologies for the obscene delay. This is a "suggested fix" for the Issue #11

### CODE REVIEW

File: DallasTemperature.cpp [hunk 1]
```diff
@@ -339,19 +339,15 @@ void DallasTemperature::requestTemperatures()
 // returns TRUE  otherwise
 bool DallasTemperature::requestTemperaturesByAddress(uint8_t* deviceAddress)
 {
+  uint8_t bitResolution = getResolution(deviceAddress);
+  if (bitResolution == 0) return false; //Device disconnected
 
   _wire->reset();
```
Comment (RobTillaart) [inline DallasTemperature.cpp]: if the root cause is in the Wire lib one can catch the return value of reset too

    int b = _wire->reset();
    if (b == 0) return false;

**Comment by RobTillaart** [comment]
@milesburton 
Did not test it but code is OK, could add extra test see above.

**Comment by milesburton** [comment]
I forget how much of a pain GitHub's PR system is. Anyway, I've created a new PR. Would you mind glancing your eye over the changes to make sure you're both happy before I merge

**Comment by milesburton** [comment]
https://github.com/milesburton/Arduino-Temperature-Control-Library/pull/34

**Comment by robtillaartfei** [comment]
I' will review all code again this week, possibly this evening

**Comment by milesburton** [comment]
Cheers. There are a few issues where the signatures have changed. I'm looking into those as soon as my lunch break kicks in

**Comment by milesburton** [comment]
Made a few modifications, but seem to be getting signature issues on the blockTillConversionComplete line. In normal C++ world I'd think this was lacking stdint, but here I can't see why the alias' arn't working. *keeps investigating*

**Pull request closed without merging**